### PR TITLE
Add AMP to Incubating Projects, update PROJECTS.md

### PR DESCRIPTION
**Pull request by brianwarner**
Welcome, AMP, to incubation status!  This PR also updates the PROJECTS.md to include an incubating section with both AMP and nvm.

Signed-off-by: Brian Warner <brian@bdwarner.com>

### CODE REVIEW

**Comment by kborchers** [comment]
I would like to request that we fast track this PR and have added the fast track label.

File: PROJECTS.md [hunk 1]
```diff
@@ -46,3 +46,9 @@ For information on the stages, please see the [Project Progression Document](/PR
 * [jQuery UI](https://jqueryui.com/)
 * [RequireJS](https://requirejs.org/)
 * [Sizzle](https://sizzlejs.com/)
+
+## Incubating Projects
+
+* [AMP](https://amp.dev/)
+* [nvm](https://nvm.sh/)
```
[the remark this answers is not in the document]
Comment (MylesBorins) [reply PROJECTS.md]: This should likely come in a different PR, unrelated to this

[the remark this answers is not in the document]
Comment (ljharb) [reply PROJECTS.md]: in this case, the line is being added here, so it should be added correctly.

**Comment by kborchers** [comment]
With 2 upvotes on the fast track request and 2+ approvals I will land this PR and will send another PR to address the nvm links

**Pull request merged**